pkg/kubelet/kubelet.go

```diff
@@ -1900,6 +1901,88 @@ func (kl *Kubelet) cleanUpContainersInPod(podId types.UID, exitedContainerID str
 	}
 }
 
+// GetPodSandboxNetNS returns the network namespace path of the given podSandbox
```

Comment:
why not? this one or the one on ip GC (https://github.com/kubernetes/kubernetes/pull/34373)? I can do both I guess, sounds like there's confusion on how to handle net plug though.